src/components/PepleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,86 @@
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types/Person';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  people: Person[],
+  selectedPerson: string,
+};
+
+export const PeopleTable: React.FC<Props> = ({ people, selectedPerson }) => {
+  const getPersonByParent = (parent: string | null) => {
+    return people.find(person => person.name === parent);
```

Comment:
It's not a good practice to leave the value undefined. Better use null here
```suggestion
    return people.find(person => person.name === parent) || null;
```